.devcontainer/reinstall-cmake.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
```

Comment:
Did not work on Windows VS Code because 'bash\r' could not be found. Just converted the file from CRLF to LF fixed it.

Reply:
Were you using a WSL setup or a non-WSL setup? WSL is probably the easier way IMO.